### ISSUE TITLE
Music not playing - Kick conversion

### ISSUE THREAD

**Issue by mikroman** [mwenge/uridium#2]
I have converted this to build in Kick Assembler. With one issue. No music. Here's my issue:
(original code)
b0E49   JSR PlaySound
        LDA a95
        STA aEF
        LDA #$01
        STA aF2
        LDA #<$FFBA
        STA a3E99
        LDA #>$FFBA
        STA a3E9A
        RTS 
Necesarry to change the pointers for the ffba loads.
lda #$00
sta $a3E99
lda #$fe
sta $a3E9A
RTS

**Comment by mwenge** [comment]
Thanks for spotting the fix for this. Out of interest, how did you figure it out?

**Comment by mikroman** [comment]
How did I find it? Heh. Lot's of hunting. The game's sound worked ; but, not the music, which is generally interrupt driven. I spotted the FFBA address earlier on, it's the ROM kernal routine SETLFS - for file system, and I just wondered why that was replacing the INT vector. How I decided what the real address was? I peeked at a working version someone had sent me to see which address was there.I had the correct spot - wasn't %100 sure of the final address; but, I suspected it was $FE00. It was. 
Thanks for doing all of the hard work. I am continuing this project and have begun the task of commenting.
